SPECS/hamcrest/hamcrest.spec

```diff
@@ -65,6 +65,7 @@ mocking libraries and UI validation rules.
 %package core
 Summary:        Core API of hamcrest matcher framework.
 Group:          Development/Libraries/Java
+Provides:       mvn(org.hamcrest:hamcrest-core)
```

Comment:
Are you sure this is necessary? Current version of `hamcrest-core` from PMC seems to already be aware of this capability:

```bash
root [ / ]# dnf -q repoquery --whatprovides "mvn(org.hamcrest:hamcrest-core)"
hamcrest-core-0:1.3-14.cm2.noarch
```